public/lib/model/format-helpers.ts

```diff
@@ -74,11 +74,28 @@ const getArticleFormatLabel = (contentType: ContentType): string => {
 }
 
 /**
- * true if the value is the label of a special format or the "Standard Article" label
+ * true if the value is the label of a list element format or the "Standard Article" label
  */
 const isFormatLabel = (value: string): boolean => {
-    return value === STANDARD_ARTICLE_FORMAT_LABEL || specialFormats.some(format => format.label === value)
+    return value === STANDARD_ARTICLE_FORMAT_LABEL || listElementArticleFormats.some(format => format.label === value)
 }
 
+const provideListElementFormatsForFilterList = (featureSwitches?: Record<string, boolean>) => {
+    return listElementArticleFormats.filter(format => !format.behindFeatureSwitch || featureSwitches[format.behindFeatureSwitch] === true)
```

Comment:
This should resolve the possibly undefined error

```suggestion
    return listElementArticleFormats.filter(format => !format.behindFeatureSwitch || featureSwitches?.[format.behindFeatureSwitch] === true)
```